src/pages/projects.tsx

```diff
@@ -0,0 +1,152 @@
+import React, { useMemo } from 'react'
+
+import Head from 'decentraland-gatsby/dist/components/Head/Head'
+import { NotMobile } from 'decentraland-ui/dist/components/Media/Media'
+import toSnakeCase from 'lodash/snakeCase'
+import Grid from 'semantic-ui-react/dist/commonjs/collections/Grid/Grid'
+
+import WiderContainer from '../components/Common/WiderContainer'
+import CurrentProjectsList from '../components/Grants/Current/CurrentProjectsList'
+import GrantsBanner from '../components/Grants/Current/GrantsBanner'
+import RequestBanner from '../components/Grants/RequestBanner'
+import BurgerMenuLayout from '../components/Layout/BurgerMenu/BurgerMenuLayout'
+import LoadingView from '../components/Layout/LoadingView'
+import MaintenanceLayout from '../components/Layout/MaintenanceLayout'
+import Navigation, { NavigationTab } from '../components/Layout/Navigation'
+import CategoryFilter, { ProjectTypeFilter } from '../components/Search/CategoryFilter'
+import StatusFilter from '../components/Search/StatusFilter'
+import {
+  OldGrantCategory,
+  ProjectStatus,
+  SubtypeAlternativeOptions,
+  SubtypeOptions,
+  toGrantSubtype,
+} from '../entities/Grant/types'
+import { toProjectStatus } from '../entities/Grant/utils'
+import { GrantWithUpdate, ProposalType } from '../entities/Proposal/types'
+import useFormatMessage from '../hooks/useFormatMessage'
+import useProjects from '../hooks/useProjects'
+import useURLSearchParams from '../hooks/useURLSearchParams'
+import { isUnderMaintenance } from '../utils/maintenance'
+
+function filterDisplayableProjects(
+  projects: GrantWithUpdate[] | undefined,
+  type: string | undefined,
+  subtype: SubtypeOptions | undefined,
+  status: ProjectStatus | undefined
+) {
+  if (!projects) {
+    return []
+  }
+
+  if (!type) {
+    return projects.filter((item) => (status ? item.status === status : true))
+  }
+
+  if (type === ProjectTypeFilter.BiddingAndTendering) {
+    return projects.filter((item) => item.type === ProposalType.Bid && (status ? item.status === status : true))
+  }
+
+  const grants = projects.filter((item) => item.type === ProposalType.Grant)
+  if (type === ProjectTypeFilter.Grants && subtype === SubtypeAlternativeOptions.Legacy) {
+    return grants.filter(
+      (item) =>
+        (status ? item.status === status : true) &&
+        Object.values(OldGrantCategory).includes(item.configuration.category as OldGrantCategory)
+    )
+  }
+
+  if (type === ProjectTypeFilter.Grants) {
+    return grants.filter(
+      (item) =>
+        (status ? item.status === status : true) &&
+        (subtype ? toSnakeCase(item.configuration.category) === toSnakeCase(subtype) : true)
+    )
+  }
+}
+
+function getCounter(projects: GrantWithUpdate[] | undefined) {
+  return {
+    all_projects: projects?.length || 0,
+    grants: projects?.filter((item) => item.type === ProposalType.Grant).length || 0,
+    bidding_and_tendering: projects?.filter((item) => item.type === ProposalType.Bid).length || 0,
```

Comment:
we could use the `ProjectCategory` here too (and the getProjectCategory to organize/filter)